packages/admin-ui/src/pages/system/actions.ts

```diff
@@ -21,21 +21,21 @@ export const changeEthClientTarget = (
   if (nextTarget === prevTarget) return;
 
   // If the previous target is package, ask the user if deleteVolumes
-  const deleteVolumes =
+  const deletePrevEthClient =
     prevTarget && prevTarget !== "remote"
-      ? await new Promise((resolve: (_deleteVolumes: boolean) => void) =>
+      ? await new Promise((resolve: (_deletePrevEthClient: boolean) => void) =>
```

Comment:
Should be equivalent to
```suggestion
      ? await new Promise<boolean>(resolve =>
```